python/lsst/ts/Dome/mock_controller.py

```diff
@@ -246,224 +238,225 @@ async def request_and_send_status(self, llc, llc_name):
+        """Set the positions of the louvers.
 
         Parameters
         ----------
-        kwargs: `dict`
-            A dictionary with arguments to the function call. It should contain the key "id" with an int
-            value and the key "position" with a float value.
+        position: array of float
+            An array of positions, in percentage with 0 meaning closed and 100 fully open, for each louver.
+            A position of -1 means "do not move".
         """
-        self.log.info(f"Received command 'setLouver' with arguments {kwargs}")
-
-        # No conversion from radians to degrees needed since both the commands and the mock az controller
-        # use radians.
-        await self.lcs.setLouver(
-            louver_id=int(kwargs["id"]), position=float(kwargs["position"])
+        self.log.info(
+            f"Received command 'setLouvers' with argument position={position}"
         )
+        await self.lcs.setLouvers(position)
 
-    async def closeLouvers(self):
+    async def close_louvers(self):
         """Close all louvers.
         """
         self.log.info(f"Received command 'closeLouvers'")
         await self.lcs.closeLouvers()
 
-    async def stopLouvers(self):
+    async def stop_louvers(self):
         """Stop the motion of all louvers.
         """
         self.log.info(f"Received command 'stopLouvers'")
         await self.lcs.stopLouvers()
 
-    async def openShutter(self):
+    async def open_shutter(self):
         """Open the shutter.
         """
         self.log.info(f"Received command 'openShutter'")
         await self.apscs.openShutter()
 
-    async def closeShutter(self):
+    async def close_shutter(self):
         """Close the shutter.
         """
         self.log.info(f"Received command 'closeShutter'")
         await self.apscs.closeShutter()
 
-    async def stopShutter(self):
+    async def stop_shutter(self):
         """Stop the motion of the shutter.
         """
         self.log.info(f"Received command 'stopShutter'")
         await self.apscs.stopShutter()
 
-    async def config(self, **kwargs):
+    async def config(self, system, settings):
         """Configure the lower level components.
 
         Parameters
         ----------
-        kwargs: `dict`
-            A dictionary with arguments to the function call. It should contain keys for all lower level
-            components to be configured with values that are dicts with keys for all the parameters that
-            need to be configured. The structure is::
-
-                "AMCS":
-                    "jmax"
-                    "amax"
-                    "vmax"
-                "LWSCS":
-                    "jmax"
-                    "amax"
-                    "vmax"
+        system: `str`
+            The name of the system to configure.
+        settings: `list` of `dict`
+            A dictionary with arguments to the function call. It should contain a key called "system" with
```

Comment:
I believe this comment needs to be updated. I think the first two sentences can be deleted, so the real description starts around "an array containing a dict".

Also please consider "an array containing a single dict" to emphasize that we really only expect one dict. (I realize this is the way EIE does it, and it's fine, but it is likely to surprise anyone who thinks you designed this API).

Reply:
Corrected, thanks!